crates/tako/src/worker/pool.rs

```diff
@@ -44,7 +44,7 @@ impl GenericResourcePool {
         }
     }
 
-    pub fn release_allocation(&mut self, allocation: GenericResourceAllocationValue) {
+    pub fn release_allocation(&mut self, allocation: &GenericResourceAllocationValue) {
```

Comment:
Why referenec was added?

It was on purpose moved so it cannot be accidentally removed twice.